prereise/gather/griddata/hifld/data_process/generators.py

```diff
@@ -132,5 +193,13 @@ def build_plant(bus, substations):
         ["Summer Capacity (MW)", "Winter Capacity (MW)"]
     ].max(axis=1)
     generators.rename({"Minimum Load (MW)": "Pmin"}, inplace=True, axis=1)
+    print("Fitting heat rate curves to EPA data... (this may take several minutes)")
```

Comment:
Can we use `progress_apply` in place of `apply` below to get a progress bar? See https://stackoverflow.com/questions/18603270/progress-indicator-during-pandas-operations

Reply:
I had tried it, but couldn't get it working, since `tqdm` was trying to import something from pandas that had been deprecated/removed.

Reply:
Too bad